main.go

```diff
@@ -57,36 +57,37 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+type cliArgs struct {
+	MetricsAddr          string `arg:"-m, --metrics-bind-address, env" default:":8080" help:"The address the metric endpoint binds to."`
+	ProbeAddr            string `arg:"-h, --health-probe-bind-address, env" default:":8081" help:"The address the probe endpoint binds to."`
+	EnableLeaderElection bool   `arg:"-l, --leader-elect, env" default:"false" help:"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager."`
+	ConfigFile           string `arg:"-c, --config-file, env" default:"/etc/spi/config.yaml" help:"The location of the configuration file."`
+	VaultInsecureTLS     bool   `arg:"-i, --vault-insecure-tls, env" default:"false" help:"Whether is allowed or not insecure vault tls connection."`
```

Comment:
I wouldn't specify `-i`, it's too specific config option to have shortcut like this.